ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge.m

```diff
@@ -138,6 +138,14 @@ - (NSDictionary *)constantsToExport
                                                         target:targetUrl] build]];
 }
 
+RCT_EXPORT_METHOD(setCustomDimensionValue:(NSString *)trackerId dimensionIndex:(NSInteger)dimensionIndex dimensionValue:(NSString *)dimensionValue)
+{
+    id<GAITracker> tracker = [[GAI sharedInstance] trackerWithTrackingId:trackerId];
+    
+    [tracker set:[GAIFields customDimensionForIndex:dimensionIndex] value:dimensionValue];
```

Comment:
I do not think it is enough to just set the dimension value, but it needs to be sent along with a screen view or event according to [the docs](https://developers.google.com/analytics/devguides/collection/ios/v3/customdimsmets#overview)